src/common/main.lua

```diff
@@ -262,13 +266,52 @@ function ant.init()
 
 		ao.send({
 			Target = msg.From,
-			-- This is out of our pattern, should be <action>-Notice
-			Action = "Reassign-Name-Submit-Notice",
+			Action = "Reassign-Name-Notice",
 			Initiator = msg.From,
 			Name = name,
 			["Process-Id"] = antProcessIdToReassign,
 		})
 	end)
+
+	createActionHandler(ActionMap.ApproveName, function(msg)
+		--- NOTE: this could be modified to allow specific users/controllers to create claims
+		utils.validateOwner(msg.From)
```

Comment:
@atticusofsparta - consider if we think allowing controllers to approve primary name requests is desired

Reply:
I have commenced developing one such module on personal projects but its still not mature.